UPGRADE-3.x.md

```diff
@@ -4,6 +4,14 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
 =======================
 
+### Deprecations
+
+`Symfony\Component\Translation\TranslatorInterface` is deprecated in favor of `Symfony\Contracts\Translation\TranslatorInterface`. You MUST replace all references to them in your code and inject correct object into:
```

Comment:
The class `Symfony\Component\Translation\TranslatorInterface` is deprecated since symfony/translation 4.2, but what we are deprecating here are the usages for this class, not the class itself:
```suggestion
Usages of `Symfony\Component\Translation\TranslatorInterface` are deprecated in favor of `Symfony\Contracts\Translation\TranslatorInterface`. You MUST replace all references to them in your code and inject correct object into:
```